### DISCUSSION TITLE
[Areabrick] Template pimcore_input("xxx") not working with {% set catId = pimcore_input("xxx") %}, but works if it's directly passed to string interpolation

### DISCUSSION THREAD

**Discussion by benyaminl** [pimcore/pimcore#17283] [General]
Hello, I'm quite new to pimcore, and I found a strange things. 

I follow the docs of https://pimcore.com/docs/platform/Pimcore/Documents/Editables/Areablock/Bricks#how-to-create-a-brick

And use 

```

{% set catId = pimcore_input("category_id") %}

{% if editmode %}
    <div>
        <h2>Product List</h2>
        <br/>
        <b>Advanced Configuration</b>
        <div>
            {{ dump(pimcore_input("category_id")) }}
            ID: {{ pimcore_input("category_id") }}
            ID: {{ catId }}
        </div>
    </div>
{% else %}
    {% if not catId.isEmpty() %}
        <h1>{{ catId.data }}</h1>

    {% endif %}
{% endif %}
```

resulting to 
![image](https://github.com/pimcore/pimcore/assets/2817332/c9eaa972-7137-49da-b4aa-7bc5fa5a8837)

But strange things when using the set variable, it only print normal string text without any input options. Is there any configuration that I need to enable? 

Thank you

**Comment by jdreesen** [comment]
You probably need to apply the `|raw` filter.

**Reply by benyaminl** [reply]
Hello @jdreesen thank you for replying to this. I tried that (pipe the variable to |raw) and still on the CMS Editor, it still render as text. It's quite strange actually. 

Is there anything else that you could guess? 

Thank you
